src/drag-and-drop/components/app.tsx

```diff
@@ -66,10 +66,38 @@ export const baseAuthoringProps = {
               title: "Image URL",
               type: "string",
               format: "uri"
+            },
+            imageWidth: {
+              title: "Image width",
+              type: "number"
+            },
+            imageHeight: {
+              title:"Image height",
+              type: "number"
+            },
+            itemLabel: {
+              title: "Label",
+              type: "string"
+            },
+            index: {
+              title: "Index",
+              type: "number"
+            },
+            itemValue: {
+              title: "Value",
+              type: "number"
+            },
+            targetMatch: {
+              title: "Target match:",
+              type: "number"
```

Comment:
What is index, itemValue and targetMatch? 
These values seem to be unused. I bet they're related to some upcoming features? Generally, I would avoiding merging that in this PR, as if the plans or schedule change, we might end up with some unused code that might be confusing. Or you might end up implementing that differently. 😉

Reply:
Index, itemValue, and targetMatch are for upcoming features. They were specified to be in this story https://www.pivotaltracker.com/story/show/178247192
ItemValue and targetMatch will be exported to the graph question interactive.

Reply:
Okay, so can we remove it from this PR and add them when they're actually used? 
Otherwise, we can end up with some unused confusing properties if the schedule or requirements change.